### ISSUE TITLE
Pause, stop or hide functionality is not provided for auto scrolling animation.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5734]
### What is the PWD impact?

Users with cognitive disabilities, low vision, vestibular disorders, or those using screen magnifiers may experience difficulty reading moving content. Continuous motion can cause distraction, nausea, or make content inaccessible.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

**Repro Steps: ​**           
1. Open URL: Web Chat in edge browser.
2. Ask query, 'Animation card'.
3. Navigate to animation.
4. Observe the issue that Pause, stop or hide functionality is not provided for auto scrolling animation.

### What do you expect?

If an animation plays automatically on the interface, Play/Pause controls should be provided to allow users to start, stop, or pause the animation as needed.

### What actually happened?

Pause, stop or hide functionality is not provided for auto scrolling animation.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/a14f8cfe-c226-4fe9-95ba-fb7c80e4856a

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide

### Adaptive Card JSON

```json

```
